src/Services/Masa.Auth.Service.Admin/Services/DepartmentService.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Masa.Auth.Service.Admin.Services;
+
+public class DepartmentService : ServiceBase
+{
+    public DepartmentService(IServiceCollection services) : base(services)
+    {
+        App.MapGet(Routing.DepartmentList, ListAsync);
+        App.MapPost(Routing.Department, CreateAsync);
+        App.MapDelete(Routing.Department, DeleteAsync);
+    }
+
+    private async Task CreateAsync([FromServices] IEventBus eventBus,
+        [FromHeader(Name = "user-id")] Guid userId, [FromBody] AddDepartmentCommand createDepartmentCommand)
```

Comment:
Maybe we need a global solution for current user info.

Reply:
It will be changed to token